js/html/shared/HTML.svelte

```diff
@@ -5,12 +5,20 @@
 	export let value: string;
 	export let visible = true;
 
-	const dispatch = createEventDispatcher<{ change: undefined }>();
+	const dispatch = createEventDispatcher<{
+		change: undefined;
+		click: undefined;
+	}>();
 
 	$: value, dispatch("change");
 </script>
 
-<div class="prose {elem_classes.join(' ')}" class:hide={!visible}>
+<!-- svelte-ignore a11y-click-events-have-key-events a11y-no-static-element-interactions -->
```

Comment:
I think this is a case where it makes sense to ignore the a11y warning since we wouldn't want to convert this entire div into a button. The developer should take care to add their own a11y based on the content of the HTML component. Does that seem reasonable @hannahblair?